src/eterna/mode/FeedbackViewMode.ts

```diff
@@ -56,14 +56,6 @@ export default class FeedbackViewMode extends GameMode {
         const background = new Background();
         this.addObject(background, this.bgLayer);
 
-        // this._puzzleTitle = Fonts.std(this._puzzle.getName(false), 14).color(0xffffff).bold().build();
-        // this._puzzleTitle.position = new Point(33, 8);
-        // this.uiLayer.addChild(this._puzzleTitle);
-
-        this._title = Fonts.std('', 12).color(0xffffff).bold().build();
```

Comment:
do we not want this anymore, or were we never really using it?

Reply:
Certainly the former, if not the latter